src/main/config/messages_template.xml

```diff
@@ -423,6 +423,11 @@ See the accompanying LICENSE file for applicable license.
     <response></response>
   </message>
 
+  <message id="DOTJ085E" type="ERROR">
+    <reason>Parameter '%1' cannot be set with param in project file. The param '%1' will be ignored.</reason>
+    <response>Use '%2' key instead.</response>
```

Comment:
I think we can simplify this to avoid repeating the parameter name, and the second sentence of the reason is sort of implied by the first. 

If we can't be more specific whether the suggested replacement is an attribute or element, I suggest we drop the noun entirely. How about this:

```suggestion
    <reason>The '%1' parameter can’t be set with &lt;param&gt; elements in project files.</reason>
    <response>Use '%2' instead.</response>
```

Reply:
@infotexture If the project file is JSON or YAML format, then `set with &lt;param&gt; elements` doesn't apply because the project file doesn't have an XML element. If we need to know what format the source file is, we need to move the validation to different class and we'd also need a different error codes for JSON, YAML, and XML. 😬

Reply:
@infotexture And because you can have a YAML project file that includes an XML project file that includes a JSON project file, we don't know what the format of the actual file that contains the param.

Reply:
Because this fix is going into a patch release, I would prefer to not refactor the project parser code to make the error messages better. That should be done in the next major/minor version. So could the copy be changed to not assume XML format?

Reply:
Oh, right, forgot we don't know which of the 3 formats we're dealing with here. 
Then I guess this is the best we can do:

```suggestion
    <reason>The '%1' parameter can’t be set with 'param' in project files.</reason>
    <response>Use '%2' instead.</response>
```

Reply:
@infotexture - I like your suggestion.